src/main/python/ttconv/isd.py

```diff
@@ -397,6 +404,64 @@ def _process_element(
 
     return None
 
+def _prune_empty_spans(element: model.ContentElement):
+  children = list(element)
+  for child in children:
+    _prune_empty_spans(child)
+    if isinstance(child, model.Text) and not child.get_text():
+      element.remove_child(child)
+    elif isinstance(child, model.Span) and not child:
```

Comment:
What do you mean by `not child` here? It cannot be `None` and instance of `Span`, right?